common/guild_base.cpp

```diff
@@ -27,8 +35,49 @@
 //until we move MAX_NUMBER_GUILDS
 #include "eq_packet_structs.h"
 
-const char *const BaseGuildManager::GuildActionNames[_MaxGuildAction] =
-{ "HearGuildChat", "SpeakGuildChat", "Invite", "Remove", "Promote", "Demote", "Set_MOTD", "War/Peace" };
+std::vector<default_permission_struct> default_permissions = {
```

Comment:
What do these values represent (the second one) ?

Reply:
Struct `DefaultPermissionStruct` vs `default_permission_struct` ?

Reply:
The GuildActionNames were the old guild system equivalent for the RoF permission system.  
The default_permissions represents the defaults for guild creation.  These were mapped from Live in summer of 2023.

Reply:
Think he just wants `default_permission_struct` changed to `DefaultPermissionStruct` for consistency in our struct nomenclature.

Reply:
Yep @Kinglykrab

Reply:
Updated